book/website/reproducible-research/open/open-hardware.md

```diff
@@ -1,111 +1,359 @@
+|Design | Developers, Contributors, Educators| Description of the overall design rationale. List of the main design choices and implications (e.g. functional compromises). Minimal explanation of the working principles, and links to external learning resources. | Development page, DEVELOPMENT.md| 
+|Technical contributions| Manufacturers, Developers, Technical support| Development roadmap, guidelines, core team and contributors, communication channels, version control system, related works. Description of the typical contribution process. Examples of (un)successful contribution processes. | Contributions page, CONTRIBUTING.md| 
+|Development| Developers| Development tools (e.g. IDEs, EDA, or CAD software) and style guidelines. Setup for development and testing. Detailed overview of the hardware modules and software. Specifications of the internal and external hardware interfaces. Tutorials and examples of how to get started. | Development page, DEVELOPMENT.md| 
+
+-->
+
+
+
+
+
+(rr-open-hardware-techdoc)=
+## Technical documents
+
+Technical documents provide the source needed to study and replicate a hardware design.
+In contrast to project documentation and community building, technical documents for OSH are quite specific, but can be considered analogous to what source code is for software.
+Depending on the project technical documents may include technical drawings, images describing electronic schematics, computer-aided design ([CAD](https://en.wikipedia.org/wiki/Computer-aided_design)) files, or assembly instructions to replicate the design.
+A thoroughly documented project will have all types of documents.
+It may also include code (firmware and software) necessary to run the hardware.
+The source files (e.g. CAD files) are best accompanied by textual and multi-medial documentation, such as guides for manufacturing, assembly, maintenance, and development.
+
+We provide here a quite exhaustive list of documentation elements.
+Your project may not need all of them, but it is worth considering having at least minimal information for most of these elements:
+
+- A **context description** which may reflects project maturity, complexity, the intentions of authors on how it should be used, and technical specifications. It may include standards compliance (the [DIN-SPEC](https://gitlab.com/OSEGermany/OHS-3105) standard for instance) and an estimation of the overall required budget and build time.
+
+- A **Bill of Materials** (BOM): a list or spreadsheet describing part numbers, putative suppliers, costs, and a short description. Make it easy to tell which item in the bill of materials corresponds to which component in your design files: use matching reference designators in both places, provide a diagram indicating which part goes where, or otherwise explain the correspondence.
```

Comment:
```suggestion
- A **Bill of Materials** (BOM): a list or spreadsheet describing part numbers, putative suppliers, costs, and a short description. 
  Make it easy to tell which item in the bill of materials corresponds to which component in your design files: use matching reference designators in both places, provide a diagram indicating which part goes where, or otherwise explain the correspondence.
```